File: RSPAClient/Core/AtPar.POCOEntities/MT_PKPL_DEVIATION_HEADER.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_PKPL_DEVIATION_HEADER
    {
        public string BUSINESS_UNIT { get; set; }
        public string ORDER_NO { get; set; }
        public int ORDER_LINE_NO { get; set; }
        public string PICK_BATCH_ID { get; set; }
        public Nullable<int> PICK_LIST_LINE_NO { get; set; }
        public Nullable<double> QTY_REQUESTED { get; set; }
        public Nullable<double> QTY_ALLOCATED { get; set; }
        public Nullable<double> QTY { get; set; }
        public int TRANSACTION_ID { get; set; }
        public string LOCATION { get; set; }
        public string INV_ITEM_ID { get; set; }
        public string DEPT_ID { get; set; }
        public Nullable<System.DateTime> PICK_DATE { get; set; }
        public Nullable<System.DateTime> UPDATE_DATE { get; set; }
        public string USER_ID { get; set; }
        public string ALLOC_UOM { get; set; }
        public string PICK_UOM { get; set; }
        public Nullable<double> CONVERSION_FACTOR { get; set; }
        public string REPORT_DATA_1 { get; set; }
        public string REPORT_DATA_2 { get; set; }
        public string REPORT_DATA_3 { get; set; }
        public string STD_UOM { get; set; }
        public Nullable<double> QTY_ORDER { get; set; }
        public string ORDER_UOM { get; set; }
        public int LINE_NO { get; set; }
        public Nullable<int> SCHED_LINE_NO { get; set; }
        public Nullable<int> DEMAND_LINE_NO { get; set; }
        public Nullable<int> SEQ_NBR { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_PKPL_DEVIATION_HEADER.ts
export class MT_PKPL_DEVIATION_HEADER {
    public BUSINESS_UNIT: string;
    public ORDER_NO: string;
    public ORDER_LINE_NO: number;
    public PICK_BATCH_ID: string;
    public PICK_LIST_LINE_NO?: number;
    public QTY_REQUESTED?: number;
    public QTY_ALLOCATED?: number;
    public QTY?: number;
    public TRANSACTION_ID: number;
    public LOCATION: string;
    public INV_ITEM_ID: string;
    public DEPT_ID: string;
    public PICK_DATE?: Date;
    public UPDATE_DATE?: Date;
    public USER_ID: string;
    public ALLOC_UOM: string;
    public PICK_UOM: string;
    public CONVERSION_FACTOR?: number;
    public REPORT_DATA_1: string;
    public REPORT_DATA_2: string;
    public REPORT_DATA_3: string;
    public STD_UOM: string;
    public QTY_ORDER?: number;
    public ORDER_UOM: string;
    public LINE_NO: number;
    public SCHED_LINE_NO?: number;
    public DEMAND_LINE_NO?: number;
    public SEQ_NBR?: number;
}